README.md

```diff
@@ -37,7 +37,7 @@ install.packages("EGRET")
     ## package 'EGRET' successfully unpacked and MD5 sums checked
     ## 
     ## The downloaded binary packages are in
-    ##  C:\Users\lcarr\AppData\Local\Temp\1\RtmpAPk7Z0\downloaded_packages
+    ##  C:\Users\lcarr\AppData\Local\Temp\1\Rtmp8QoZAB\downloaded_packages
```

Comment:
what do you think, should we keep in the messages (it's kind of nice to see what the real output looks like)? Also a sneaky way to make sure people are rebuilding the readme